datasets/modeled/ind_residuals/dataset.json5

```
@@ -0,0 +1,122 @@
+{
+  dataset_id: "decarb_2023_ind_residuals",
+  dataset_type: "modeled",
+  sector_description: "Industrial",
+  description: "Industrial residual load profiles for the contiguous US for DECARB 2023 High Scenario, calculated using industrial and eia861 projected data for 2024.",
+  origin_creator: "Ashreeta Prasanna",
+  origin_organization: "NREL",
+  origin_contributors: [
+    "Ashreeta Prasanna",
+    "Elaine Hale",
+    "Daniel Thom",
+    "Lixi Liu"
+  ],
+  origin_project: "DECARB",
+  origin_date: "2024-07-09",
+  origin_version: "v0.1.0",
+  data_source: "EIA, NREL, IGATE-E",
+  source: "Industrial residual calculation for DECARB 2023",
```

```suggestion
  source: "NREL",
```

I'm just mimicking the type of information we gave for the industrial dataset (https://github.com/dsgrid/dsgrid-project-DECARB/blob/main/datasets/modeled/industry/dataset.json5#L17). Apologies that the name of this metadata field is ambiguous ...

---

LBNL and ORNL are not involved in residual calculations